docs/reference/plugins.md

```diff
@@ -1 +1,18 @@
 # Plugins
+
+Shift supports the integration of custom Plugins. Shift Plugins are Qt based UIs which can interact with Shift to display additional information, perform actions or modify the behaviour of the application.
+
+## Shift Native Plugins
+
+Shift ships with two plugins which can be found in the *Tools* section in the top menu bar. The desired Plugin can be opened by clicking on the corresponding entry.
+
+Once a Plugin is opened, the Shift UI can be customized with the best suited layout.
+
+The *Operators List* Plugin helps us visualize the list of operators in the workflow and shows the their current progress.
+
+![Opening a Plugin](/images/opening_a_plugin.gif)

```

Comment:
Please, check how to add the html code from this card and apply it to this image: https://github.com/Inbibo/shift_docs/issues/45

Ensure it renders properly on the webpage.

Reply:
changed the link to html format and it works